docker-compose.yml

```diff
@@ -3,5 +3,9 @@ version: "3"
 services:
   server-status:
     image: mgerb/server-status:latest
+    build:
```

Comment:
I think overall the changes in `Dockerfile` are okay. I believe the changes in this file can be removed because the compose script doesn't build the image. `image` and `build` in this case are redundant.